spec/loader/shared_print_loader_spec.rb

```diff
@@ -68,4 +69,22 @@
 
     it_behaves_like "valid commitment with expected fields"
   end
+
+  describe "when saving policies to db" do
+    it "saves as an array" do
+      # Setup, load commitments from fixture
+      Cluster.collection.find.delete_many
+      fixt = fixture "sp_commitment_policies.tsv"
+      Loader::FileLoader.new(batch_loader: described_class.for(fixt))
+        .load(fixt, filehandle: described_class.filehandle_for(fixt))
+      # Get policies on all loaded commitments
+      all_policies = Cluster.each.map do |cluster|
+        cluster.commitments.map do |commitment|
+          commitment.policies.class
+        end
+      end
+      # Check that the loaded policies were saved as arrays
+      expect(all_policies.flatten.uniq).to eq [Array]
```

Comment:
This was confusing to me, probably because `all_policies` has the _class_ of each policy rather than the actual policy. 

The intention might be clearer with this:

```ruby
all_policies = Cluster.each.map do |cluster|
  cluster.commitments.map(&:policies)
end

expect(all_policies.flatten).to all ( be_a(Array) )
```

Reply:
Yeah that's better and less confusing. Implemented.